src/actions/multiple_target_action.py

```diff
@@ -0,0 +1,47 @@
+from ..primitive_target import STRICT_HERE
+from talon import Context, Module
+
+mod = Module()
+
+ctx = Context()
+ctx.matches = r"""
+tag: user.cursorless
+"""
+
+
+mod.list("cursorless_multiple_target_action", desc="Cursorless move or bring actions")
+ctx.lists["self.cursorless_multiple_target_action"] = {"bring", "move", "swap", "call"}
```

Comment:
Unified the syntax for all these multi-target commands. Slightly awkward for some of them but not too bad so far; will have to see how well it works for a couple days

Reply:
I total understand why you did it but I'm not quite happy with it :s